examples/graphics/render-asset.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>PlayCanvas Asset Render</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <link rel="icon" type="image/png" href="../playcanvas-favicon.png" />
+    <script src="../../build/playcanvas.js"></script>
+    <script src="../../build/playcanvas-extras.js"></script>
+    <style>
+        body {
+            margin: 0;
+            overflow-y: hidden;
+        }
+    </style>
+</head>
+
+<body>
+    <!-- The canvas element -->
+    <canvas id="application-canvas"></canvas>
+
+    <!-- The script -->
+    <script>
+        var canvas = document.getElementById("application-canvas");
+
+        // Create the app and start the update loop
+        var app = new pc.Application(canvas);
+
+        // Set the canvas to fill the window and automatically change resolution to be the same as the canvas size
+        app.setCanvasFillMode(pc.FILLMODE_FILL_WINDOW);
+        app.setCanvasResolution(pc.RESOLUTION_AUTO);
+
+        window.addEventListener("resize", function () {
+            app.resizeCanvas(canvas.width, canvas.height);
+        });
+
+        var miniStats = new pcx.MiniStats(app);
+        app.start();
+
+        // load cube glb file
+        var cubeEntities = [];
+        app.assets.loadFromUrl("../assets/models/playcanvas-cube.glb", "container", function (err, asset) {
+
+            // get the instance of it set up with render component and add it to scene
+            cubeEntities[0] = asset.resource.instantiateRenderEntity();
+            cubeEntities[0].setLocalPosition(7, 12, 0);
+            cubeEntities[0].setLocalScale(3, 3, 3);
+            app.root.addChild(cubeEntities[0]);
+
+            // get another instance of it set up with render component and add it to scene
+            cubeEntities[1] = asset.resource.instantiateRenderEntity();
```

Comment:
Is this faster/better than doing `cubeEntities[1] = cubeEntities[0].clone();`?

Reply:
It's matching the other case here .. performance should be similar.
But I've changed it to .clone() to test this path .. and that exposed an issue .. I'll have separate PR for this in a minute.